docs/snippets/angular/your-component.ts.mdx

```diff
@@ -10,7 +11,8 @@ export default {
   component: YourComponent,
 } as Meta;
 
-const Template: Story<YourComponent> = (args) => ({
+const Template: Story<YourComponent> = (args:YourComponent) => ({
```

Comment:
This doesn't look right to me. It should be something more like:

```
const Template: Story<YourComponentProps> =  (args: YourComponentProps) => ({ ... })
```

However, I'm not sure where the `YourComponentProps` type comes from